source/renderer/app/stores/AddressesStore.js

```diff
@@ -47,7 +47,10 @@ export default class AddressesStore extends Store {
       if (address != null) {
         this._refreshAddresses();
         runInAction('set last generated address and reset error', () => {
-          this.lastGeneratedAddress = address;
+          this.lastGeneratedAddress = new WalletAddress({
```

Comment:
@tomothespian why do we have to use `new WalletAddress` here? Isn't the address already  of `WalletAddress` domain?